user/customizing-the-build.md

```diff
@@ -114,7 +114,15 @@ Currently, neither the `after_success` nor `after_failure` have any influence on
 
 ## Deploying your Code
 
-An optional phase in the build lifecycle is deployment. This step can't be overridden, but is defined by using one of our continuous deployment providers to deploy code to Heroku, Engine Yard, or a different supported platform.
+An optional phase in the build lifecycle is deployment. This step can't be
+overridden, but is defined by using one of our continuous deployment providers
+to deploy code to Heroku, Engine Yard, or a different supported platform.
+
+When deploying files to a provider, prevent Travis CI from deleting all files
```

Comment:
I'd make this more explicit. Something like: "By default Travis CI resets your working directory (by running `git stash --all`), removing all changes that potentially have been made while running your tests or other build steps. In order to prevent this, and for example keep files generated as part of you build you can add the `skip_cleanup` option to your configuration: